.github/workflows/package.yaml

```diff
@@ -18,15 +18,16 @@ jobs:
   package:
     strategy:
       matrix:
-        os: [macos-intel, macos-arm, windows, linux]
         include:
-        - os: macos-intel
+        - platform: mac
```

Comment:
Maybe add a comment that the `platform` keys must be valid `electron-builder` platform options, and cannot be freely chosen (the `arch` keys however are not related to electron and are just used for filename tags).